lib/purescript-cst/purescript-cst.cabal

```diff
@@ -3,7 +3,7 @@ cabal-version: 2.4
 name:           purescript-cst
 version:        0.1.1.0
 synopsis:       PureScript Programming Language Concrete Syntax Tree
-description:    The surface syntax of the PureScript Programming Language.
+description:    The surface and underlying syntax of the PureScript Programming Language.
```

Comment:
If we change the README according to my comment there, let's make the same change here.

Reply:
Done.